test/controllers/PdfControllerSpec.scala

```diff
@@ -108,6 +114,9 @@ class PdfControllerSpec extends SpecBase {
               )
 
               getSourceString(result) mustEqual responseBody
+
+              verify(mockAuditService).audit(eqTo(IF_DATA_RECEIVED), eqTo(Some(trustJson)))(any(), any())
```

Comment:
I don't think Alison wanted the data in the event audited out as we can get this from the outbound requests to IF already.